src/fix/main.c

```diff
@@ -746,6 +747,15 @@ static const uint8_t ninLogo[] = {
 	0xDD, 0xDC, 0x99, 0x9F, 0xBB, 0xB9, 0x33, 0x3E
 };
 
+static const uint8_t trashLogo[] = {
+	0xFF^0xCE, 0xFF^0xED, 0xFF^0x66, 0xFF^0x66, 0xFF^0xCC, 0xFF^0x0D, 0xFF^0x00, 0xFF^0x0B,
```

Comment:
`~` instead of `0xFF^` doesn't pass `-Woverflow`.

Reply:
It might with suffix `hh`, but I doubt that'd be worth the effort.

Reply:
"`h`" or "`hh`" are not [valid suffixes](https://en.cppreference.com/w/c/language/integer_constant). I thought they might be GNU extensions, but GCC 10.2 doesn't compile them. Also I think that `~ X` would promote X to an integer, as would `~ (uint8_t) X`; you'd need `(uint8_t) ~ X` to not do that.